src/snowflake/connector/ocsp_snowflake_async.py

```diff
@@ -0,0 +1,416 @@
+#
```

Comment:
dumb question: inside this module, other than the async call, are the other logics the same as the sync one?

Reply:
Yep, any changes are internal for async compatibility and should not change behaviour externally.

Reply:
nice!